arches/app/search/search_export.py

```diff
@@ -61,10 +61,9 @@ def __init__(self, search_request=None):
         self.set_precision = GeoUtils().set_precision
 
     def insert_subcard_below_parent_card(self, main_card_list, sub_card_list):
-        for sub_card in sub_card_list:
-            parent_obj = sub_card.nodegroup.parentnodegroup_id
-            for main_card in main_card_list:
-                if main_card.nodegroup_id == parent_obj:
+        for main_card in main_card_list:            
+            for sub_card in sub_card_list:
+                if main_card.nodegroup_id == sub_card.nodegroup.parentnodegroup_id:
```

Comment:
Just checking that the `for` iterator on `main_card_list` picks up sub cards inserted into it on line 68?

Reply:
Absolutely, `main_card_list` is added to at index+1, so `sub_card` processed on next subsequent iteration.

(don't like changing current loop list on the fly, but minor refactor for bugfix,, o_0)

Reply:
ok